HeterogeneousCore/CUDAServices/interface/CUDAService.h

```diff
@@ -15,23 +15,35 @@ namespace edm {
   class ParameterSet;
   class ActivityRegistry;
   class ConfigurationDescriptions;
-}
+}  // namespace edm
 
 namespace cudaserviceimpl {
   template <typename T>
-  struct make_device_unique_selector { using non_array = cudautils::device::unique_ptr<T>; };
+  struct make_device_unique_selector {
+    using non_array = cudautils::device::unique_ptr<T>;
+  };
   template <typename T>
-  struct make_device_unique_selector<T[]> { using unbounded_array = cudautils::device::unique_ptr<T[]>; };
+  struct make_device_unique_selector<T[]> {
+    using unbounded_array = cudautils::device::unique_ptr<T[]>;
+  };
   template <typename T, size_t N>
-  struct make_device_unique_selector<T[N]> { struct bounded_array {}; };
```

Comment:
OK, I don't understand any more the logic... why some one-liners are split, while some trivial multi-line functions are put on a single line ?

Reply:
It could be related to these being templates. Is the "trivial multi-line function put on a single line" you refer to a template or not?